### ISSUE TITLE
Incorrect identification of commented tags

### ISSUE THREAD

**Issue by enkessler** [r-cochran/cuke_sniffer#95]
The following feature file is triggering the 'Scenario has a commented out tag' rule on Scenario Bar, even though there are no commented tags:

```
Feature: FooBar

  Scenario: Foo
    * a step

  # Just a comment
  @some_tag
  Scenario: Bar
```

The same feature with no preceding scenarios does not trigger the issue:
```
Feature: FooBar


  # Just a comment
  @some_tag
  Scenario: Bar
```

**Comment by enkessler** [comment]
It looks like #85 might fix this.
